src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -256,4 +260,14 @@ public ResponseEntity<List<String>> getUserInfo(
         return ResponseEntity.ok(userInformationService.getUserCaseRoles(caseId, authorization));
     }
 
+    @PostMapping(path = "/{caseId}/courtDecision")
+    @Operation(summary = "Get decision on proposed claimant repayment")
+    public RepaymentDecisionType getDecisionOnClaimantProposedRepayment(
```

Comment:
The method name getDecisionOnClaimantProposedRepayment suggests that it's a getter method, but it's actually performing an action. Consider renaming it to something like calculateDecisionOnClaimantProposedRepayment to better reflect its purpose.